src/ButtonGroup.py

```diff
@@ -14,8 +14,12 @@ def add(self, button):
 
     def display(self, screen):
         """Draw all Button objects in the ButtonGroup."""
-        for _, button in self.buttons.items():
-            button.display(screen)
+        try:
```

Comment:
This would need to be fixed, hiding exceptions like this is generally a bad idea